lib/routes.js

```diff
@@ -105,6 +105,11 @@ export default class Routes {
 
         result += `:${this.id}`;
     }
+    
+    switch (action) {
+      case 'index':
+        result = addTrailingSlash(result);
+    }
```

Comment:
Why not putting this switch/case inside the one above?
Why not doing the same with the one below?

```js
    switch (action) {
      case 'new':
      case 'show':
      case 'edit':
        result = addTrailingSlash(result);
        result += action;
      case 'update':
      case 'destroy':
        result = addTrailingSlash(result);
        result += `:${this.id}`;
      case 'index':
        result = addTrailingSlash(result);
    }
```